test/TestConfig/configure.pl

```diff
@@ -34,23 +34,35 @@
 my $machineConfiguremk = $configuredir . "/machineConfigure.mk";
 unlink($machineConfiguremk);
 my $platform=`uname`;
-if ($platform !~ /Linux/i) {
-	exit;
-}
+
+my $ZOSR14=3906;
 my %capability_Hash;
 
-# virtual infor
-my $KVM_Image=`cat /proc/cpuinfo | grep -i QEMU`;
-my $VMWare_Image=`lspci 2>/dev/null | grep -i VMware` ;
-my $Virt_Sys="";
+if ($platform =~ /OS\/390/i) {
+
+	my $hardware=`uname -m`;
+	if ($hardware >= $ZOSR14) {
+		$capability_Hash{guardedstorage} = 1;
+	}
```

Comment:
This isn't necessarily true. We can very well be on a z14 but guarded storage disabled due to missing OS support. I guess it's a decent starting point though but we ought to be checking the port library or something for these kinds of queries. This shouldn't block the merge of this PR but something to think about for the future.